library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/focals/CirclePromptFocal.java

```diff
@@ -101,6 +110,9 @@ public void prepare(@NonNull PromptOptions options, float targetX, float targetY
         mBounds.top = targetY - mBaseRadius;
         mBounds.right = targetX + mBaseRadius;
         mBounds.bottom = targetY + mBaseRadius;
+
+        mPath = new Path();
+        mPath.addCircle(mPosition.x, mPosition.y, mBaseRadius, Path.Direction.CW);
```

Comment:
Woops, a bit of an oversight by myself here, this should be in the update method. Now the prompt focus isn't pulsing :(